.github/workflows/ci-e2e-tests.yml

```diff
@@ -0,0 +1,178 @@
+name: E2E tests
+
+permissions:
+  id-token: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - main
+    tags:
+      - "v*.*.*"
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  e2e_tests:
+    name: E2E tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v3
+      
+      - name: Install Nix
+        uses: cachix/install-nix-action@v27
+        # relax sandbox due impure kontrol-service tests during build
+        with:
+          extra_nix_config: |
+            sandbox = relaxed
```

Comment:
just for learning purposes, What is this for?

Reply:
This was copied from the kardinal kontrol CI tests and it is needed there because the build runs some tests pulling teh plugins from Github.  It is not needed here.